point_test.go

```diff
@@ -105,7 +117,7 @@ var _ = Describe("Point", func() {
 
 		for i := 0; i < trials; i++ {
 			a = RandomPoint()
-			axf, ayf = a.XY()
+			axf, ayf, _ = a.XY()
```

Comment:
Consider explicitly confirming that there is no error.

Reply:
Fixed in bf8a281.